COPYRIGHT

```diff
@@ -1,9 +1,16 @@
-Percona server for PostgreSQL
+Percona Server for PostgreSQL - License
 (formerly known as Postgres, then as Postgres95)
 
-Portions Copyright (c) 2024-2025, Percona LLC and/or its affiliates
+This software includes portions copyrighted or made available by:
+•	Percona, LLC and/or its affiliates (collectively, “Percona”)
```

Comment:
I'm not sure about the use of these unicode characters (“ and ”) instead of the simple normal double quote ", as otherwise this is a plain ascii file.